### DISCUSSION TITLE
Forking a/are WM(s) and customizing it for Parch

### DISCUSSION THREAD

**Discussion by mmdbalkhi** [parchlinux/.github#4] [General]
Please provide your suggestions, I will your suggests add to this message

- [qtile](https://github.com/qtile/qtile)
- [pywm](https://github.com/jbuchermn/pywm)
- [newm](https://github.com/jbuchermn/newm)

**Comment by behdanisohrab** [comment]
Pywm(has been discontinued but i think it's worth reviving)

**Reply by mmdbalkhi** [reply]
I added to the message but a large part of the Pywm is written in C, maybe we not work on it
